repos/system_upgrade/el7toel8/actors/bindupdate/actor.py

```diff
@@ -0,0 +1,34 @@
+from leapp.actors import Actor
+from leapp.models import BindFacts, InstalledRedHatSignedRPM
+from leapp.tags import PreparationPhaseTag, IPUWorkflowTag
+from leapp.libraries.actor import updates
+
+
+class BindUpdate(Actor):
+    """
+    Actor parsing facts found in configuration and modifing configuration.
+    """
+
+    name = 'bind_update'
+    consumes = (InstalledRedHatSignedRPM, BindFacts)
+    produces = ()
+    tags = (PreparationPhaseTag, IPUWorkflowTag)
+
+    pkg_names = {'bind', 'bind-sdb', 'bind-pkcs11'}
+
+    def has_package(self, t_rpms):
+        """Replacement for broken leapp.libraries.common.rpms.has_package."""
```

Comment:
Hmm it should not be broken, and if it is - we (the leapp team) should fix it as it's a library function. Is there an open issue?

Reply:
No issue known, I think this function were copied originally from checkpostfix actor. Not sure anymore, I will test using proper library.